packages/runtime/src/ctx.ts

```diff
@@ -329,6 +333,20 @@ export class Ctx implements Context {
 
 		this.verifyTEALVersionIsMatch(approvalProgTEAL, clearProgTEAL);
 
+		//verify that MaxGlobalSchemaEntries <= 64 and MaxLocalSchemaEntries <= 16
+		const globalSchemaEntries = flags.globalInts + flags.globalBytes;
+		const localSchemaEntries = flags.localInts + flags.localBytes;
```

Comment:
What happen when localInts(or other value) is negative number? 
I also don't think use will pass negative number, but better we should check it too.

Reply:
It already throws an error when there is a negative number passed

Reply:
Thanks.